app/facades/profiles/edit_facade.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Profiles
+  class EditFacade
+    attr_reader :profile
+
+    delegate :id, to: :profile, prefix: true
+    delegate :user, to: :profile, prefix: true
+
+    def initialize(profile)
+      @profile = profile
+    end
+
+    def user_name
+      profile_user.name
```

Comment:
you can also delegate this. if you need / want to have `user_name` method, you can delegate like this:
`delegate :name, to: :profile_user, prefix: :user`. (Or you can delegate user to profile without prefix, it's clear what this means)